guides/micronaut-metrics-aws/micronaut-metrics-aws.adoc

```diff
@@ -0,0 +1,43 @@
+external:micronaut-metrics/start.adoc[]
+
+common:aws.adoc[]
```

Comment:
FYI: If you want the What you will need section to look the same as OCI see https://github.com/micronaut-projects/micronaut-guides/blob/master/guides/micronaut-cloud-database-aws/micronaut-cloud-database-aws.adoc